tests/integration/search/test_factories_search.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+import pytest
+from elasticsearch_dsl import Search
+from mock import MagicMock, patch
+
+from inspirehep.search.api import InspireSearch, LiteratureSearch
+from inspirehep.search.factories.search import (
+    _get_search_with_source,
```

Comment:
I feel like this is tested directly, although it's an internal/private method (`_`)